tests/integration/Cron/Workers/SendingQueue/SendingThrottlingHandlerTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace MailPoet\Test\Cron\Workers;
+
+use MailPoet\Cron\Workers\SendingQueue\SendingThrottlingHandler;
+use MailPoet\Settings\SettingsController;
+
+class SendingThrottlingHandlerTest extends \MailPoetTest {
```

Comment:
I think we should add test cases for increasing batch size after certain amount of successes and maybe also that the lowest possible batch size is 1.